src/sulguk/transformer.py

```diff
@@ -64,12 +64,12 @@ def _find_attr(
     def _get_classes(self, attrs: Attrs):
         return self._find_attr("class", attrs).split()
 
-    def _get_a(self, attrs: Attrs) -> Entity:
-        url = self._find_attr("href", attrs)
-        if url:
-            return Link(url=url)
-        else:
-            return Group()
+    def _get_url(self, attrs: Attrs) -> Entity:
+        for name in ('href', 'src'):
```

Comment:
`href` is valid only for `a`
while `src` is expected for `img`

let's split into two functions

Reply:
thank you! done in d706bdcab6c365bed66a18858c294783d522162b